src/components/chart-index.js

```diff
@@ -0,0 +1,62 @@
+import { useSelector } from "react-redux";
+import { Sparklines, SparklinesLine, SparklinesReferenceLine } from "react-sparklines";
+import _ from 'lodash';
+
+const ChartIndex = () => {
+  
+  const chart = useSelector((state) => state.chart)
+
+  const renderChart = () => {
+    
+    const renderSparkline = (data, dataType, color, unit) => {
+      const parsedWeatherData = data.map(list => list.main[dataType]);
+      const average = _.round(_.sum(parsedWeatherData)/parsedWeatherData.length);
+      
+      return (
+        <>
+          <Sparklines data={parsedWeatherData} width={150} svgHeight={150}>
+            <SparklinesLine color={color} />
+            <SparklinesReferenceLine type="avg" />
+          </Sparklines>
+          <h4 className="text-center">{average}{unit}</h4>
+        </>
+      )
+    }
+    
+    return chart.map((row, i) => (
+      <tr key={i}>
+        <th scope="row" className="h4 align-middle">{row.data.city.name}</th>
+        <td>{renderSparkline(row.data.list, 'temp', 'orange', '°F')}</td>
```

Comment:
Nice work here making `renderSparkline` reusable for each data type

Reply:
Thanks! I realized I was about to copy and paste three functions, but I realized I could make it reusable instead.